### PR TITLE
MCKIN-10868 - Update Image Explorer schema via background task

### PR DESCRIPTION
**Pull request by naeem91**
All Image Explorer instances authored with pixel values needs to be converted to percentage values. This PR adds a management command which runs a celery task to update schema in Mongo modulestore. 
The task can be run for a single course or all the open courses in the system.

### CODE REVIEW

File: edx_solutions_api_integration/tasks.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+
+def _convert_pixel_to_percentage(width, height, x_in_pixel, y_in_pixel):
+    x_in_percentage = (float(x_in_pixel) / width) * 100
+    y_in_percentage = (float(y_in_pixel) / height) * 100
+
+    return '{}%'.format(x_in_percentage), '{}%'.format(y_in_percentage)
+
+
+def _save_updated_block(self, block):
```
Comment (msaqib52) [inline edx_solutions_api_integration/tasks.py]: please remove this unused function.

File: edx_solutions_api_integration/tasks.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    except ImportError:
+        return url
+
+    url = '"{}"'.format(url)
+    lms_relative_url = replace_static_urls(url, course_id=course_id)
+    lms_relative_url = lms_relative_url.strip('"')
+    return _make_url_absolute(lms_relative_url)
+
+
+def _make_url_absolute(url):
```
Comment (msaqib52) [inline edx_solutions_api_integration/tasks.py]: can we instead use [prefix_with_lms_base](https://github.com/edx-solutions/api-integration/blob/master/edx_solutions_api_integration/utils.py#L400) utility method?

File: edx_solutions_api_integration/tasks.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    try:
+        img = Image.open(urllib.urlopen(image_url))
+    except Exception as e:
+        logger.warning('Failed loading image: {}'.format(e.message))
+        return None
+    else:
+        return img.size
+
+
+def _replace_static_from_url(url, course_id):
```
Comment (msaqib52) [inline edx_solutions_api_integration/tasks.py]: I think we should move it to [utils.py](https://github.com/edx-solutions/api-integration/blob/master/edx_solutions_api_integration/utils.py)

File: edx_solutions_api_integration/tasks.py [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+        if img_size:
+            width, height = img_size
+
+    if width and height:
+        width, height = _convert_pixel_to_percentage(width, height, hotspot.get('x'), hotspot.get('y'))
+        hotspot.set('x', width)
+        hotspot.set('y', height)
+
+
+def _get_image_dimensions(image_url):
```
Comment (msaqib52) [inline edx_solutions_api_integration/tasks.py]: I think we should move it to [utils.py](https://github.com/edx-solutions/api-integration/blob/master/edx_solutions_api_integration/utils.py)

File: edx_solutions_api_integration/tasks.py (hunk 3 above)
Comment (naeem91) [reply edx_solutions_api_integration/tasks.py]: this is a one time script, we can remove it once it complete

**Comment by naeem91** [comment]
@msaqib52 review changes has been made

**Pull request merged**